packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -163,6 +163,16 @@ html.interface-interface-skeleton__html-container {
 	@include break-medium() {
 		display: flex;
 	}
+
+	.block-editor-block-breadcrumb {
```

Comment:
Not really clear why this is in the interface package indeed. We can move it to the breadcrumb component directly.
The other thing is that it has a lot of unnecessary stuff. Why it has a background, z-index, ...